src/components/list/themes/light/list-header.indigo.scss

```diff
@@ -0,0 +1,8 @@
+@use './list-header.base.scss';
```

Comment:
This should rather be imported and used as the default static styles on the list-header.ts. It shouldn't be imported here.